gitea/release.go

```diff
@@ -46,6 +47,33 @@ func (c *Client) GetRelease(user, repo string, id int64) (*Release, error) {
 	return r, err
 }
 
+// ListReleaseAttachments gets all the assets of a release in a repository
+func (c *Client) ListReleaseAttachments(user, repo string, id int64) (*Release, error) {
```

Comment:
This should return a `[]*Attachment`, not `*Release` https://developer.github.com/v3/repos/releases/#list-assets-for-a-release